pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -254,37 +270,54 @@ func (r *ReconcileWindowsMachineConfig) deleteWindowsVMs(count int) bool {
 	return true
 }
 
+func (r *ReconcileWindowsMachineConfig) createWindowsVM() (types.WindowsVM, error) {
+	// Create Windows VM in the cloud provider
+	log.V(1).Info("creating a Windows VM")
+	createdVM, err := r.cloudProvider.CreateWindowsVM()
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating windows VM")
+	}
+	log.V(1).Info(fmt.Sprintf("created the Windows VM: %s", createdVM.GetCredentials().GetInstanceId()))
+	return createdVM, nil
+}
+
+func (r *ReconcileWindowsMachineConfig) createWindowsWorkerNode(vm types.WindowsVM) error {
```

Comment:
I am again not sure what you get with this helper. Moreover this seems like it is configuring the Windows VM.